pkg/rancher-desktop/backend/containerClient/nerdctlClient.ts

```diff
@@ -280,28 +336,7 @@ export class NerdctlClient implements ContainerEngineClient {
 
       console.log('ran nerdctl compose up', result);
     } finally {
-      await this.runCleanups(cleanups);
-    }
-  }
-
-  async stop(container: string, options?: ContainerStopOptions): Promise<void> {
-    function addNS(...args: string[]) {
-      if (options?.namespace) {
-        return [`--namespace=${ options.namespace }`, ...args];
-      }
-
-      return args;
-    }
-
-    if (options?.delete && options.force) {
-      await this.nerdctl(...addNS('container', 'rm', '--force', container));
-
-      return;
-    }
-
-    await this.nerdctl(...addNS('container', 'stop', container));
-    if (options?.delete) {
-      await this.nerdctl(...addNS('container', 'rm', container));
+      this.runCleanups(cleanups);
     }
   }
 
```

Comment:
We don't need to copy the compose file into the VM for doing `nerdctl compose down` do we? IIRC with `docker-compose` you need access to the compose file to do `docker-compose down`.

Reply:
Good catch, the compose files are now copied over in b4987fd8236cf99b83d8545c35aa135927103d3e (it didn't make any sense to pass host-side `composeDir` in either way).